src/NuGet.Status/App_Code/ViewHelpers.cshtml

```diff
@@ -92,9 +92,9 @@
     // The Usabilla button is not displayed on the status page.
 }
 
-@helper CookieComplianceScript()
+@helper CookieComplianceScript(IHtmlString cspNonce)
 {
-    // The cookie compliance script is not needed on the status page.
+    <script src="https://wcpstatic.microsoft.com/mscc/lib/v2/wcp-consent.js" nonce="@cspNonce"></script>
```

Comment:
@zhhyu I was talking with @agr about this change and we have some concerns about using this script on the Status page. What does the script do exactly and how will it affect users in for example Europe? How can I test this?